apps/frontend/src/app/2_molecules/WalletIdentity/WalletIdentity.module.css

```
@@ -0,0 +1,3 @@
+.address-badge {
+  @apply inline-flex items-center bg-gray-50 h-8 font-roboto text-gray-10 text-xs font-semibold px-2.5 leading-none select-none rounded;
```

```suggestion
  @apply inline-flex items-center bg-gray-50 h-8 font-roboto text-gray-10 text-xs font-semibold px-2 leading-none select-none rounded;
```
in design, I see 8px padding from the left and right sides

---

the design is not correct, cause this component should have the same padding as dropdown does. Will ask gilad to update it

---

ah, got it